test-clients/src/main/java/com/hedera/services/yahcli/README.md

```diff
@@ -0,0 +1,95 @@
+# Yahcli v0.0.1
+A command line interface that is able to perform the following actions against a specified network.
+
+1. Account Operations
+    1. Get Balance.
+2. System File Operations
+    1. Download all/specific file.
+    2. Upload a System file.
+3. Fees Operations
+    1. List Basic Transaction and Query fees of all/specific service.
+
+## Getting Started
+### Setting up your environment
+
+Create a folder structure like this.<br>
```

Comment:
Better to mention the location of this folder structure if it needs to be at the top level `hedera-services` or in `test-clients`

Reply:
This folder structure can be made anywhere. You just have to run the docker command from the base of this folder structure.

Reply:
Got it. thanks Anirudh !